wavefront_sdk/common/heartbeater_service.py

```diff
@@ -0,0 +1,63 @@
+"""
+Service that periodically reports component heartbeats to Wavefront.
+
+@author Hao Song (songhao@vmware.com).
+"""
+
+import time
+import logging
+from threading import Timer
+from wavefront_sdk.common.constants import HEART_BEAT_METRIC, \
+    APPLICATION_TAG_KEY, CLUSTER_TAG_KEY, SERVICE_TAG_KEY, SHARD_TAG_KEY, \
+    COMPONENT_TAG_KEY, NULL_TAG_VAL
+
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=too-few-public-methods
+class HeartbeaterService(object):
```

Comment:
Aren't we adding any usage examples for the HeartbeaterService into README?

Reply:
This will be only used by our own up-tier SDKs.